build_product_images.py

```diff
@@ -39,6 +45,7 @@ def parse_args():
     )
     parser.add_argument("-u", "--push", help="Push images", action="store_true")
     parser.add_argument("-d", "--dry", help="Dry run.", action="store_true")
+    parser.add_argument("-a", "--architecture", help="Target platform for image, Expecting -a <platform 1> <platform 2> ... At least one argument", nargs='+', required=True)
```

Comment:
`buildx` doesn't work out of the box for users not using docker-desktop and has to be installed separately. Maybe just add a line in the description to make that clear, so it reads like e.g.: "Build and publish product images. See conf.py for details regarding product versions.You may need to install buildx as a cli-plugin if not using docker-desktop: see the Docker site for details."

Reply:
Done.